out.txt

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0"?>
```

Comment:
is this a file that needs to be checked in? This looks like something generated

Reply:
That was a whoopsie. Fixed